src/server/pps/server/api_server.go

```diff
@@ -1002,6 +1020,21 @@ func (a *apiServer) DeletePipeline(ctx context.Context, request *pps.DeletePipel
 	}); err != nil {
 		return nil, err
 	}
+
+	// Delete output repo
+	if request.DeleteRepo {
+		pfsClient, err := a.getPFSClient()
+		if err != nil {
+			return nil, err
+		}
+		if _, err := pfsClient.DeleteRepo(ctx, &pfs.DeleteRepoRequest{
+			Repo:  &pfs.Repo{request.Pipeline.Name},
+			Force: true,
```

Comment:
Doesn't this mean that we could delete a repo that another pipeline is waiting on for changes? That seems like it might be a problem.